README.md

```diff
@@ -311,6 +311,23 @@ See `docs/SWTPM.md`
 
 See `docs/WindowTBS.md`
 
+## Building using CMake
+
+CMake supports compiling in many environments including Visual Studio
+if CMake support is installed. The commands below can be run in
+`Developer Command Prompt`.
+
+```
+mkdir build
+cd build
+# to use installed wolfSSL location (library and headers)
+cmake .. -DWITH_WOLFSSL=/prefix/to/wolfssl/install/
+# OR to use a wolfSSL source tree
+cmake .. -DWITH_WOLFSSL_TREE=/path/to/wolfssl/
```

Comment:
Using `/path/to/wolfssl/` work for me... was missing `wolfssl/options.h`. However I was able to use `/usr/local`.

Reply:
Not able to reproduce any error building. Maybe older wolfTPM headers installed in the same path as wolfSSL.

Reply:
Ah, I was not using `WITH_WOLFSSL_TREE`. However then I tried it and got:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    WITH_WOLFSSL_TREE
```

Then deleted the build dir and started over and it worked fine. Saw a bunch of configure like messages `-- Looking for arpa/inet.h` and `-- Looking for arpa/inet.h - found` and all built great.